### DISCUSSION TITLE
Learn Cluster API Videos/Streams

### DISCUSSION THREAD

**Discussion by richardcase** [kubernetes-sigs/cluster-api#9227] [Ideas]
I was thinking of kicking-off a number of live streams on Cluster API, specifically focussed on its usage (and not its development).

The idea is that we can gradually build up a catalogue of videos that people can use as a learning resource.

Each episode will be:

- short, say 20 - 30 mins max, on a specific topic so that they are easily digestible and not too onerous to prepare for. 
- Some theory, demo heavy (but this may vary depending on the topic).
- Use StreamYard (i can supply the account)

Some initial ideas for topics:

| Name               | Description |
| ------------------ |  ----------- |
| Getting Started    | Introduction and guide the first time user to creating a cluster                           |
| Cluster Operations | Performing operations against an existing cluster. Scaling, upgrading, adding new machines |
| CAPA Overview      | A look at the AWS provider, its capabilities and getting started                           |
| CAPZ Overview      | A look at the Azure provider, its capabilities and getting started                         |
| CAPV Overview      | A look at the vSphere provider, its capabilities and getting started                       |
| CAPI & GitOps      | How to do GitOps with Cluster API                                                          |
| ClusterClass       | A look at how to use author ClusterClasses and use them                                    |
| Autoscaling        | Look at how to autoscale a cluster managed by CAPI                                         |
| CAPX Overview | A look at the Nutanix provider, its capabilities and getting started   |

What other topcs do we need? Backup, moving a management cluster? Feel free to suggest topics.

A couple of asks:

- Is this a good idea & useful? Any suggestions?
- What additional topics would you like to see?
- Would you be willing to be the expert guest for an episode? (without this help this will be a non-starter)
- Would you like to co-host?
- Do you have artistic skills? If so, we could do with some branding images.

**Comment by sbueringer** [comment]
Sounds like a great idea!

Not much free bandwidth at the moment, totally overloaded with reviews.. :/

**Comment by cprivitere** [comment]
I would be interested in helping/co-hosting.

**Comment by dtzar** [comment]
I'd be happy to create the CAPZ videos and the GitOps ones.  The GitOps one is related to [the issue assigned to me](https://github.com/kubernetes-sigs/cluster-api/issues/8479).

**Comment by elmiko** [comment]
i agree this sounds like a nice idea, i'd be willing to co-host a session on cluster autoscaling if someone wants to join.

**Comment by tuxtof** [comment]
Very good initiative @richardcase can you add CAPX to the list

**Reply by richardcase** [reply]
Thanks @tuxtof ....added to the list.

**Comment by richardcase** [comment]
Better late than never, lets get this kicked off in the next few weeks. I'll start the process.

**Reply by jsakil14** [reply]
I am looking for k8s node upgrades using CAPI...just one video suggestion

**Comment by NMSVishal** [comment]
Great idea ,I would be interested in helping.

**Reply by richardcase** [reply]
Thanks for the reminder with this, keeps falling off my todo list. I'll add it to the list of topics for the office hours next week.
